### ISSUE TITLE
Add auto-updating library list with library tally count

### ISSUE THREAD

**Issue by tekktrik** [adafruit/CircuitPython_Community_Bundle#118]

**Comment by TheKitty** [comment]
Is this still an issue?

**Comment by tekktrik** [comment]
Yes, I think the idea was to generate the same library download stats here as is done for the Adafruit Bundle, which hasn't been implemented yet.

**Comment by tekktrik** [comment]
Sorry, I misspoke - the idea was to have an updating list of libraries when they are added as submodules, though still similar to the one the Adafruit Bundle has:  https://github.com/adafruit/Adafruit_CircuitPython_Bundle/blob/main/circuitpython_library_list.md
